src/Webhooks/Events/OnLeave.cs

```diff
@@ -8,7 +9,14 @@ public class OnLeave : IScript
         [Target(GameSourceEvent.PlayerDestroy, ExecutionMode.Event)]
         public void OnEvent(ShPlayer player)
         {
-            Core.Instance.joinWebhook.Send(string.Format(Core.Instance.Settings.Server.PlayerLeaveFormat, player.username), player.username);
+            List<Embed> leaveEmb = new List<Embed>();
+
+            foreach (var em in Core.Instance.Settings.Server.PlayerLeaveEmbed)
+            {
+                leaveEmb.Add(new Embed { Title = string.Format(em.Title, player.username), Description = string.Format(em.Description, player.username) });
+            }
+            Core.Instance.joinWebhook.Send(string.Format(Core.Instance.Settings.Server.PlayerLeaveFormat, player.username), player.username,
+                embeds: Core.Instance.Settings.Server.PlayerLeaveUseEmbed? leaveEmb:null);
```

Comment:
Seems like this is a lot of duplicate code, maybe we would have it in a function that takes in the embed settings and returns it back:
  ```cs
public List<Embed> CreateAllEmbeds(List<Embed> embeds) {
            var emb = new List<Embed>();
            foreach (var em in Core.Instance.Settings.Server.PlayerLeaveEmbed)
            {
                emb.Add(new Embed { Title = string.Format(em.Title, player.username), Description = string.Format(em.Description, player.username) }); // add more things to format and make docs on what each mean in the json
            }
}
 ```